index.html

```diff
@@ -1,76 +1,200 @@
-              <p class="alt">Build your mobile apps in the cloud with pipelines.</p>
-              <p>Build mobile apps in an isolated, scalable and deterministic way in the cloud.</p>
-              <p><a href="/digger/" class="btn btn-primary">Learn about AeroGear<strong>Digger</strong></a></p>
-          </div><!-- col -->
-
-
-        </div><!-- row -->
-      </div><!-- container -->
-    </section><!--main features-->
-
-
+<section class="main-features">
+  <div class="container">
+    <h2 class="section-header">AeroGear Services</h2>
+
+    <div class="row">
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/mobile-ci-cd/"><i class="fa fa-cogs"></i> CI CD</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/mobile-ci-cd/" class="btn btn-primary">Learn about <strong>CI CD</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/metrics/"><i class="fa fa-bar-chart"></i> Metrics</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/metrics/" class="btn btn-primary">Learn about <strong>Metrics</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/push/"><i class="fa fa-external-link"></i> Push</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/push/" class="btn btn-primary">Learn about <strong>Push</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/keycloak/"><i class="fa fa-lock"></i> Identity Management</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/keycloak/" class="btn btn-primary">Learn about <strong>Identity Management</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/crc/"><i class="fa fa-plug"></i> CRC</a>
```

Comment:
small thing: I'd replace `CRC` with `Custom Runtime Connector` as a title, since CRC is an internal acronym